src/utils/NumberUtils.h

```diff
@@ -99,7 +99,12 @@ really_inline from_chars_result from_chars(const char *first, const char *last,
 
     float
 #ifdef _WIN32
+#if defined(__MINGW32__) || defined(__MINGW64__)
```

Comment:
Alternatively, should I make this check in the CMakeLists file, so that this is safely reverted to `_strtof_l` once the mingw-w64 platform updates their SDK?

Reply:
Ideally it would be a test directly in the source code if possible? Can be done later in any case I think given the current compilation issue.